src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Figure {
```

Comment:
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```
abstract class doesn't have to override methods